apps/ichat-gw/Server.cxx

```diff
@@ -751,7 +751,7 @@ Server::checkSubscription(const std::string& to, const std::string& from)
    msg.addArg("sendSubscriptionResponse");
    msg.addArg(from.c_str());
    msg.addArg(to.c_str());   
-   msg.addArg(Data((unsigned long)success).c_str());   
```

Comment:
looking at where ```sendSubscriptionResponse``` is handled in JabberComponent.cxx it appears to expect a numeric type too.  Some of these types were changed in Data.hxx constructors a while ago but the ichat-gw code was never updated, thanks for fixing this.